solidity_deposit_contract/README.md

```diff
@@ -23,3 +23,12 @@ Install the latest version of `dapp` by following the instructions at [dapp.tool
 ```sh
 make test_deposit_contract
 ```
+## Verifying bytecode
+
+The following parameters were used to generate the bytecode for the `DepositContract` available in this repository:
+
+* Contract Name: `DepositContract`
+* Compiler Version: Solidity `v0.6.11+commit.5ef660b1`
+* Optimization Enabled: `Yes` with `5000000` runs
```

Comment:
Wouldn't it be better to directly provide the solc commandline options?
```
solc --optimize --optimize-runs 5000000 --metadata-literal --bin deposit_contract.sol
```

Reply:
Yeah, maybe I'm the wrong one to do this PR. I don't know how this was imported from @axic 's repository. Having the Makefile around was certainly helpful.

I can add the solc cli args to the README too for now.